site/content/entry/yuzu-progress-report-feb-2023/index.md

```diff
@@ -0,0 +1,381 @@
+
+Solving this oopsie resolves crashes happening in `Metroid Prime Remastered`, and most likely others too.
+
+## UI changes
+
+The LDN lobby window got some love from newcomer [unfamiliarplace](https://github.com/unfamiliarplace), adding the option to {{< gh-hovercard "9713" "hide empty rooms." >}}
+Nothing beats quality of life changes like this, thanks!
+
+{{< imgs
+	"./lobby.png| Only the dankest rooms, please"
+  >}}
+
+For Discord gamers, [SoRadGaming](https://github.com/SoRadGaming) gives us proper game images for {{< gh-hovercard "9720" "Discord Status!" >}}
+The images are grabbed from our compatibility wiki, which is under a rewrite, so expect some games to be missing for now.
+
+{{< single-title-imgs
+    "Make your friends envy you with this simple trick!"
+    "./discord1.png"
+    "./discord2.png"
+    >}}
+
+A small mistake caused the web applet to lose the ability to redraw and zoom its content when resizing the yuzu web applet window.
+{{< gh-hovercard "9761" "Re-adding" >}} the `setZoomFactor` call fixes the scaling, allowing players to fit their Mario Manual to their needs.
+
+Per-game settings are very useful, but also very tricky to get right codewise.
+[m-HD](https://github.com/m-HD) greets us by adding a few missing graphical settings to the list, solving conflicting issues when setting the fullscreen mode, resolution scaling, filter, and antialiasing values in the {{< gh-hovercard "9784" "per-game" >}} configuration window.
+
+German77 then properly implemented per-game configuration support for the `Force maximum clocks` {{< gh-hovercard "9863" "setting" >}} AMD GPUs [benefit](https://yuzu-emu.org/entry/yuzu-progress-report-jan-2023/) so much from.
+
+While we are constantly working to improve the user experience, yuzu sometimes does crash, that’s expected of any first generation emulator.
+What shouldn’t happen is missing changes made to the configuration after a crash, or a forced close.
+The problem was in our Qt UI, and german77 worked to properly tell Qt to save and sync the configuration to file, avoiding any {{< gh-hovercard "9817" "configuration change loss" >}} after a force close.
+
+## General fixes
+
+[MonsterDruide1](https://github.com/MonsterDruide1) continues to do wonders with the [LDN](https://yuzu-emu.org/entry/ldn-is-here/) code.
+This time, proper error handling.
+One of the possible errors when handling TCP connections is `ECONNRESET`, which happens when the other end closes the connection abruptly.
+Oldschool gamers call this “rage quitting”.
+
+Jokes aside, when a sudden disconnection like this happens, the error used to get caught by yuzu instead of the game, causing the game to never get the notification of the client disconnection.
+Properly {{< gh-hovercard "9843" "passing the error" >}} through to the game allows it to handle the issue on its own way instead of crashing the yuzu LDN session.
```

Comment:
```suggestion
Properly {{< gh-hovercard "9843" "passing the error" >}} through to the game allows it to handle the issue its own way instead of crashing the yuzu LDN session.
```